scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
```

Comment:
I'll admit, I use `//` in my personal projects using SCSS.  I think we should be consistent with our use of comments structure.  Either use all `//` or `/* */`, or else define specific use cases for each version.

Advantage of `//` is it is similar to our JavaScript style guide, and could help promote consistent commenting styling across all JS and SCSS files. http://contribute.jquery.org/style-guide/js/#comments

That being said, if we're trying to make sure our SCSS stays as close to valid CSS has possible, then we should stick to `/* */`